js/huobi.js

```diff
@@ -3031,13 +3031,17 @@ module.exports = class huobi extends Exchange {
         //
         await this.loadMarkets ();
         const request = {};
+        const market = this.market (symbol);
+        let method = 'contractPublicGetLinearSwapApiV1SwapHistoricalFundingRate';
+        if (market['inverse']) {
+            method = 'contractPublicGetSwapApiV1SwapHistoricalFundingRate';
+        }
         if (symbol === undefined) {
```

Comment:
We already receive this error if symbol is undefined, so there's no need to write an explicit check for undefined symbols here

```
% huobi fetchFundingRateHistory                           
2021-12-09T03:40:54.406Z
Node.js: v14.17.0
CCXT v1.63.19
huobi.fetchFundingRateHistory ()
BadSymbol huobi does not have market symbol undefined
---------------------------------------------------
[BadSymbol] huobi does not have market symbol undefined

    at market                     ../../ccxt/ccxt/js/base/Exchange.js:979  throw new BadSymbol (this.id + ' does not have market symbol ' + symbol)
    at fetchFundingRateHistory    ../../ccxt/ccxt/js/huobi.js:3033         const market = this.market (symbol);                                    
    at processTicksAndRejections  internal/process/task_queues.js:95                                                                               
    at async main                 ../../ccxt/ccxt/examples/js/cli.js:246   const result = await exchange[methodName] (... args)                    

huobi does not have market symbol undefined
```